android/build.gradle

```diff
@@ -38,6 +38,7 @@ repositories {
 dependencies {
     //noinspection GradleDynamicVersion
     api 'com.facebook.react:react-native:+'
-    implementation 'com.revenuecat.purchases:purchases:3.1.0'
+    implementation 'com.revenuecat.purchases:purchases:3.1.1'
```

Comment:
we still need one because purchases-hybrid-common doesn't expose the dependency transitively, we can change it in the next version of the common library